apps/test/unit/lib/kits/maker/boards/microBit/AccelerometerTest.js

```diff
@@ -1,6 +1,6 @@
 import {expect} from '../../../../../../util/reconfiguredChai';
 import Accelerometer from '@cdo/apps/lib/kits/maker/boards/microBit/Accelerometer';
-import {MicrobitStubBoard} from '../makeStubBoard';
+import {MBFirmataClientStub} from '../../../../../../../src/lib/kits/maker/util/makeStubBoard';
```

Comment:
When I used the path '@cdo/lib/kits/maker/util/makeStubBoard', I received the following error:
Thus, I used this relative path instead. 
![Screen Shot 2023-04-05 at 4 31 55 PM](https://user-images.githubusercontent.com/107423305/230216393-ab3866a7-02f4-4aff-aae5-88a8acac86bf.png)

Reply:
I may be remembering wrong, but I think the issue is that it should be 
`@cdo/apps/lib/kits/maker/util/makeStubBoard`
rather than
`@cdo/lib/kits/maker/util/makeStubBoard`

Reply:
Thanks!